src/ElevatorSubsytem/Elevator.java

```diff
@@ -11,23 +11,33 @@
 import Networking.Receivers.DMA_Receiver;
 import Networking.Transmitters.DMA_Transmitter;
 import com.sun.jdi.InvalidTypeException;
+import Configuration.Configurator;
+import Configuration.Config;
 
 import java.util.HashMap;
 import java.util.Set;
 
 public class Elevator implements Runnable {
     /** Single floor travel time */
-    private static final long TRAVEL_TIME = 8500;
-    private  static final long LOAD_TIME = 1000;
     private final int elevNum;
     private int currentFloor;
     @Deprecated
     private Direction direction;
+    private long travelTime;
+    private long loadTime;
     private final HashMap<DestinationEvent, Integer> passengerCountMap;
     private final DMA_Transmitter transmitterToScheduler;
     private final DMA_Receiver receiver;
 
     public Elevator(int elevNum, DMA_Receiver receiver, DMA_Transmitter transmitter) {
+
+        // Configure Elevator from JSON
```

Comment:
This should probably stay static, maybe raise an issue and we refactor it later. Idk if we'll make a singleton or a separate config class / object to import.